### ISSUE TITLE
mini_librispeech nooverlap DER is much higher than overlap DER

### ISSUE THREAD

**Issue by shengzhang0222** [espnet/espnet#5234]
I run the mini_librispeech egs with SA-EEND model, and the number of speakers in both training and test sets is limited to 2. 
When i simulate overlap data, the DER is 29.28, it's same as reported.

But When i simulate nooverlap data, the DER is 52.79.
---------------------------------------------
SCORED SPEAKER TIME = 106005.40 secs (100.0 percent of scored speech)
MISSED SPEAKER TIME =  14388.07 secs ( 13.6 percent of scored speaker time)
FALARM SPEAKER TIME =   6600.66 secs (  6.2 percent of scored speaker time)
 SPEAKER ERROR TIME =  34976.58 secs ( 33.0 percent of scored speaker time)
SPEAKER ERROR WORDS =      0         (100.0 percent of scored speaker words)
---------------------------------------------
 OVERALL SPEAKER DIARIZATION ERROR = 52.79 percent of scored speaker time  `(ALL)

**Comment by sw005320** [comment]
Can you elaborate on your explanations more?
Do you mean that you played with the simulation part, e.g., https://github.com/espnet/espnet/blob/master/egs2/mini_librispeech/diar1/local/data.sh#L16 ?

**Comment by shengzhang0222** [comment]
Thanks for your reply.

yes, i set simu_opts_overlap=no in data.sh, but get DER=52.79. The parameters are as follows:

# simulation options
simu_opts_overlap=no
simu_opts_num_speaker=2
simu_opts_sil_scale=2
simu_opts_rvb_prob=0.5
simu_opts_num_train=500

In theory, the DER will be lower than simulating overlap data ?

**Comment by sw005320** [comment]
> In theory, the DER will be lower than simulating overlap data ?

If training works well, yes, but I'm not sure.
Can you paste the learning curves?
Also, can you count how many people appeared during inference?
The dominant errors come from the SPEAKER ERROR, and I'm curious whether the two speakers are correctly predicted.

**Comment by shengzhang0222** [comment]
I recently deleted the related experiment data, I will rerun the experiment and give you feedback.

**Comment by shengzhang0222** [comment]
![IMG_export_20230630_105442309](https://github.com/espnet/espnet/assets/41928867/9f16a0b6-6118-49fe-864e-5d63ce0364c3)
![IMG_export_20230630_105439328](https://github.com/espnet/espnet/assets/41928867/fc7c009e-de8c-4274-8165-5a5a14280785)
![IMG_export_20230630_105435804](https://github.com/espnet/espnet/assets/41928867/4a4e0a7a-ae61-4004-bd94-f91e67dce28a)
@sw005320 I rerun the experiment, and the learning curves are listed above.
The test set and training set are generated in the same way, and the number of speakers is also 2.

**Comment by sw005320** [comment]
Thanks!
Could you also paste the same figures with the overlapped case?
I want to check the comparisons.

**Comment by shengzhang0222** [comment]
@sw005320 the figures of overlapped experiment are as follows: 

![IMG_export_20230703_141118416.jpg](https://github.com/espnet/espnet/assets/41928867/b36da936-6d26-44b6-9e6f-e3c3d8400ac5)![IMG_export_20230703_141114355.jpg](https://github.com/espnet/espnet/assets/41928867/5fc451c5-ed14-4da8-af39-767b035c6ed6)![IMG_export_20230703_141110352.jpg](https://github.com/espnet/espnet/assets/41928867/35e1891d-ee95-489a-b30c-5eb102ab111f)![IMG_export_20230703_141106502.jpg](https://github.com/espnet/espnet/assets/41928867/745dfc33-ff83-4864-be02-ffc741c9951f) 

please help check, thanks.

**Comment by sw005320** [comment]
Thanks a lot!
By considering the DER value, it seems that the model was not trained well in the non-overlap case.
It might have some bugs, but can you tune some hyper-parameters (e.g., learning rate) to see whether the learning curve (especially DER) is improved?

**Comment by shengzhang0222** [comment]
Yes, of course.
The parameters I use are default parameters (conf/train_diar.yaml). 
Could you suggest how to modify the hyper-parameters?

**Comment by sw005320** [comment]
Please tune https://github.com/espnet/espnet/blob/master/egs2/mini_librispeech/diar1/conf/train_diar.yaml#L32

**Comment by shengzhang0222** [comment]
OK， I will try and give you feedback.

**Comment by shengzhang0222** [comment]
@sw005320 I conducted several comparative experiments on the learning rate, and the DERs were all above 50.

LR 0.05: 
MISSED SPEAKER TIME =   9581.78 secs (  9.0 percent of scored speaker time)
FALARM SPEAKER TIME =   9004.67 secs (  8.5 percent of scored speaker time)
 SPEAKER ERROR TIME =  35115.19 secs ( 33.1 percent of scored speaker time)

LR 0.005:
MISSED SPEAKER TIME =  13176.84 secs ( 12.4 percent of scored speaker time)
FALARM SPEAKER TIME =   7110.29 secs (  6.7 percent of scored speaker time)
 SPEAKER ERROR TIME =  35805.07 secs ( 33.8 percent of scored speaker time)

LR 0.001:
MISSED SPEAKER TIME =  12287.87 secs ( 11.6 percent of scored speaker time)
FALARM SPEAKER TIME =   8737.51 secs (  8.2 percent of scored speaker time)
 SPEAKER ERROR TIME =  36247.05 secs ( 34.2 percent of scored speaker time)

**Comment by sw005320** [comment]
Thanks a lot.
It might be either of
1. Modeling capacity
2. Big

@Jungjee, can we discuss it?

**Comment by Jungjee** [comment]
@sw005320 , Sure! 

As a start how about going through audiofile-level DERs and see how the files with the worst DERs are like?
Do you think you can share few (e.g., 3) audiofiles with worst DERs for both w/ and w/o overlap? 

Also can you share which value among FA, Miss, and SC has increased the most for DER changing from 29% to 5x%?

**Comment by Jungjee** [comment]
Also I would recommend you to hear some samples for `without overlap` case. Sometimes this really helps :-)
